### PR TITLE
Suggestions for some UI/UX improvements

### PR DESCRIPTION
**Pull request by JohaSem**
**This pull request contains**:
* Replacement of padding with proper margin (from previous `20dp` padding to `16dp` margin)
* A ripple effect on user click on `news_item`'s (if approved, it is highly recommended to add this to all other clickable items)
* Removes unessecary `card_elevation` according to style guides in material design for lists (this is optional ofcourse)

### CODE REVIEW

File: app/src/main/res/layout/news_list_row.xml [hunk 1]
```diff
@@ -1,34 +1,26 @@
...
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_gravity="center"
         card_view:cardBackgroundColor="?attr/cardViewBackgroundColor"
-        card_view:cardCornerRadius="3dp"
-        card_view:cardElevation="6dp"
-        card_view:cardUseCompatPadding="true"
-        card_view:contentPaddingLeft="20dp"
-        card_view:contentPaddingRight="20dp"
-        card_view:contentPaddingTop="4dp">
```
Comment (JohaSem) [inline app/src/main/res/layout/news_list_row.xml:17]: This padding needs to be replaced with margins and moved to the rootlayout if you want a nice touch feedback

**Comment by gustavaa** [comment]
Hi @JohaSem and thank you for this PR. I do like the ripple effect that this adds to the news cards, but I have a few thoughts:

- Personally I prefer the more defined shadow that the `card_elevation` attribute adds to the card layout. I know 1dp is the default value for cards, is this what you meant by saying that `card_elevation` is unnecessary or do you have link to this section in the material guidelines?
- I also personally prefer a little bit more margin between the cards and the screen borders, does the material guidelines say anything about this?
- Finally I think that if we were to change the look of the cards and add a ripple effect we should do it everywhere for consistency, so maybe it would be better to create a reusable card layout that can be used everywhere through the `<include>` tag.

**Comment by JohaSem** [comment]
> 
> 
> Hi @JohaSem and thank you for this PR. I do like the ripple effect that this adds to the news cards, but I have a few thoughts:
> 
>     * Personally I prefer the more defined shadow that the `card_elevation` attribute adds to the card layout. I know 1dp is the default value for cards, is this what you meant by saying that `card_elevation` is unnecessary or do you have link to this section in the material guidelines?
> 
>     * I also personally prefer a little bit more margin between the cards and the screen borders, does the material guidelines say anything about this?
> 
>     * Finally I think that if we were to change the look of the cards and add a ripple effect we should do it everywhere for consistency, so maybe it would be better to create a reusable card layout that can be used everywhere through the `<include>` tag.

I agree with the consistency part, my initial thought was to showcase how it would look with the touchfeedback on the `CurrentNewsListFragment`.
The new commits gives the same touch feedback effect on the other list items aswell. 
I reverted the `cardElevation` if you prefer it at `6dp`.

**Comment by gustavaa** [comment]
> > Hi @JohaSem and thank you for this PR. I do like the ripple effect that this adds to the news cards, but I have a few thoughts:
> > ```
> > * Personally I prefer the more defined shadow that the `card_elevation` attribute adds to the card layout. I know 1dp is the default value for cards, is this what you meant by saying that `card_elevation` is unnecessary or do you have link to this section in the material guidelines?
> > 
> > * I also personally prefer a little bit more margin between the cards and the screen borders, does the material guidelines say anything about this?
> > 
> > * Finally I think that if we were to change the look of the cards and add a ripple effect we should do it everywhere for consistency, so maybe it would be better to create a reusable card layout that can be used everywhere through the `<include>` tag.
> > ```
> 
> I agree with the consistency part, my initial thought was to showcase how it would look with the touchfeedback on the `CurrentNewsListFragment`.
> The new commits gives the same touch feedback effect on the other list items aswell.
> I reverted the `cardElevation` if you prefer it at `6dp`.

I see! Well, nice work anyway. I'll merge this now. Thank you!

**Pull request merged**